src/Microsoft.Health.Dicom.Core/DicomCoreResource.Designer.cs

```diff
@@ -87,15 +87,6 @@ internal class DicomCoreResource {
             }
         }
         
```

Comment:
Just double-check that this file has been successfully regenerated

Reply:
sure